src/Ting/Repository/Metadata.php

```diff
@@ -122,12 +125,12 @@ public function getDatabase()
      * Set repository name
      * @param string $className
      * @return $this
-     * @throws \CCMBenchmark\Ting\Exception
+     * @throws SyntaxException
      */
     public function setRepository($className)
     {
-        if (substr($className, 0, 1) === '\\') {
-            throw new Exception('Class must not start with a \\');
+        if ('\\' === ($className[0] ?? '')) {
```

Comment:
Do we really want yoda style condition in the codebase ? I'm not a fan of it personally.

Reply:
I actually did it because I'm used to Yoda style, but to the project's coding standards we should stick.
What's the rule about this in the project?

BTW, we may want to add PHP-CS-Fixer to help keeping the CS in the project.

Reply:
After a quick look, we don't seem to use yoda style.